README.md

```diff
@@ -1,54 +1,65 @@
-
-For any security issues see [`SECURITY.md`](https://github.com/Kentico/.github/blob/main/SECURITY.md).
+# Xperience by Kentico: Sitefinity Migration Tool
+
+[![Kentico Labs](https://img.shields.io/badge/Kentico_Labs-grey?labelColor=orange&logo=data:image/svg+xml;base64,PHN2ZyBjbGFzcz0ic3ZnLWljb24iIHN0eWxlPSJ3aWR0aDogMWVtOyBoZWlnaHQ6IDFlbTt2ZXJ0aWNhbC1hbGlnbjogbWlkZGxlO2ZpbGw6IGN1cnJlbnRDb2xvcjtvdmVyZmxvdzogaGlkZGVuOyIgdmlld0JveD0iMCAwIDEwMjQgMTAyNCIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPjxwYXRoIGQ9Ik05NTYuMjg4IDgwNC40OEw2NDAgMjc3LjQ0VjY0aDMyYzE3LjYgMCAzMi0xNC40IDMyLTMycy0xNC40LTMyLTMyLTMyaC0zMjBjLTE3LjYgMC0zMiAxNC40LTMyIDMyczE0LjQgMzIgMzIgMzJIMzg0djIxMy40NEw2Ny43MTIgODA0LjQ4Qy00LjczNiA5MjUuMTg0IDUxLjIgMTAyNCAxOTIgMTAyNGg2NDBjMTQwLjggMCAxOTYuNzM2LTk4Ljc1MiAxMjQuMjg4LTIxOS41MnpNMjQxLjAyNCA2NDBMNDQ4IDI5NS4wNFY2NGgxMjh2MjMxLjA0TDc4Mi45NzYgNjQwSDI0MS4wMjR6IiAgLz48L3N2Zz4=)](https://github.com/Kentico/.github/blob/main/SUPPORT.md#labs-limited-support) [![CI: Build and Test](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/actions/workflows/ci.yml/badge.svg)](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/actions/workflows/ci.yml)
+
+## Description
+
+This repository is part of the [Xperience by Kentico Migration Toolkit](https://github.com/Kentico/xperience-by-kentico-migration-toolkit).
+
+The Sitefinity Migration Tool transfers content and other data from **Sitefinity** to **Xperience by Kentico**.
+
+**Note:** This project currently falls under **Kentico Labs limited support** but will transition to **Full support by 7-day bug-fix policy** [when v1.0.0 is released](https://roadmap.kentico.com/c/218-sitefinity-migration-tool).
+
+## Screenshots
+
+[![Sitefinity Migration tool migration paths](/images/xperience-by-kentico-migration-toolkit-sitefinity-migration-tool-embedded.jpg)](/images/xperience-by-kentico-migration-toolkit-sitefinity-migration-tool-embedded.jpg)
+
+## Library Version Matrix
+
+| Xperience Version | Sitefinity Version | UMT Version | Library Version |
+| ----------------- | ------------------ | ----------- | --------------- |
+| >= 28.2.0         | 14.4.x             | 1.2.0       | 1.0.0           |
+
+### Dependencies
+
+- [ASP.NET Core 8.0](https://dotnet.microsoft.com/en-us/download)
+- [Xperience by Kentico](https://docs.kentico.com/changelog)
+- [Progress Sitefinity CMS 14.4](https://www.progress.com/documentation/sitefinity-cms/144/sitefinity-cms-documentation)
+- [XbyK Universal Migration Tool](https://github.com/Kentico/xperience-by-kentico-universal-migration-tool)
+
+## Quick Start
+
+1. Download this package from the repository [Releases](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/releases)
+1. Check XbyK and Sitefinity settings to ensure proper communication. See [Usage Guide](./docs/Usage-Guide.md) for details.

```

Comment:
If you still need to read the usage guide to setup the integration, I'd just omit the quick start and keep the "Full instructions" as "Setup instructions"

Reply:
This has been resolved. I have removed the Quick Start and updated Full Instructions to Setup Instructions